x/swaprouter/types/routes.go

```diff
@@ -9,22 +9,22 @@ import (
 	gammtypes "github.com/osmosis-labs/osmosis/v13/x/gamm/types"
 )
 
-// AccountI defines the account contract that must be fulfilled when
-// creating a x/swaprouter keeper.
+// AccountKeeper defines the account contract that must be fulfilled when
+// creating a x/gamm keeper.
 type AccountI interface {
 	NewAccount(sdk.Context, authtypes.AccountI) authtypes.AccountI
 	GetAccount(ctx sdk.Context, addr sdk.AccAddress) authtypes.AccountI
 	SetAccount(ctx sdk.Context, acc authtypes.AccountI)
 }
 
-// BankI defines the banking contract that must be fulfilled when
-// creating a x/swaprouter keeper.
+// BankKeeper defines the banking contract that must be fulfilled when
+// creating a x/gamm keeper.
 type BankI interface {
```

Comment:
These should say AccountI and BankI as well as x/swaprouter I believe

Reply:
Ugh, I keep merging directly merging from `concentrated-liqudity-main` that has an incorrect state. Sorry for the same issues again